server/controllers/finance/vouchers.js

```diff
@@ -260,6 +263,23 @@ function create(req, res, next) {
   // preprocess the items so they have uuids as required
   items.forEach(value => {
     let item = value;
+    // Only for Employee Salary Paiement
+    if (voucherType === 7) {
+      if (item.document_uuid) {
+        const updatePaiement = `
```

Comment:
This SQL statement does not have anything to do with the loop - it could easily be moved out, like this:
```js
const updateSQL = ` some sql`

items.forEach(value => {
  updatesPaiementData.push({query : updateSQL, params : [ /* ... */ ] });
});
```